src/training/CMakeLists.txt

```diff
@@ -306,6 +307,7 @@ target_link_libraries       (text2image
     ${PangoCairo_LIBRARIES}
     ${PangoFt2_LIBRARIES}
     ${FontConfig_LIBRARIES}
+    ${Glib_LIBRARIES}
```

Comment:
Those libraries are normally already part of `${Pango_LIBRARIES}`. Please verify why this is not the case for you.